frontend/src/components/Room.tsx

```diff
@@ -151,6 +152,7 @@ class Room extends React.Component<Props, State> {
     const { id } = this.props.match.params;
     this.socket.on(Event.CONNECT, () => {
       this.socket.emit(Event.JOIN_ROOM, id);
+      this.socket.emit(Event.CREATE_USERNAME, this.props.location.state.username);
```

Comment:
What about the case where someone joins the room by pasting in the URL? Would this create an empty username automatically? Maybe we should generate some name or something in that case. Would be good if we could change the generated name but this can be a later feature.

Reply:
Oops didn't see the anonymous generation you put in `server.ts` disregard this.